### ISSUE TITLE
EWC forms display incorrectly when desktop scaling is enabled

### ISSUE THREAD

**Issue by dyavc** [Dyalog/ewc#50]
EWC forms are truncated on the right hand side and I need to widen them by hand

Maybe it’s just my computer, but all the demo forms come up slightly short on the right, so I need to widen the form as things near the very right edge are truncated.

19.0.50074 64-bit Unicode on MS Windows 10

'.' ⎕wg 'devcaps'
1080 1920  194 344  ¯1 125

]link.create # d:/git/ewc

EWC.FOLDER←'d:\git\ewc\'
      demo.Run'Desktop'

![EWCdemoformScreenshot](https://github.com/user-attachments/assets/67825797-d648-4a85-b437-195a4fd7fe14)

**Comment by mkromberg** [comment]
I can reproduce something similar by setting the Windows desktop scaling to something other than 100% AND checking "Enable DPI Scaling of the GUI Application" in the Dyalog configuration. Can you confirm your settings?

**Comment by dyavc** [comment]
Yes, My Windows desktop scaling is 125%.  I have "Enable DPI Scaling of the interpreter and development environment" ticked, but not the "Enable DPI Scaling of the GUI Application".

Now, If I do tick "Enable DPI Scaling of the GUI Application", and have both ticked, I no longer have the issue, and the EWC demo forms are fine.

**Comment by sloorush** [comment]
Related to https://github.com/Dyalog/ewc-client/issues/393#issuecomment-3522376713

Will investigate
